lib/gh/remote.rb

```diff
@@ -75,7 +75,8 @@ def generate_response(key, response)
 
     # Internal: ...
     def http(verb, url, headers = {}, &block)
-      connection.run_request(verb, url, nil, headers, &block)
+      body = headers.delete :body
```

Comment:
This is somewhat iffy; ideally, we would be changing the call signature of `#http`, so that `body` is set properly without really hacking into `headers` parameter.